app/src/main/java/org/oppia/android/app/customview/VerticalDashedLineView.kt

```diff
@@ -0,0 +1,63 @@
+package org.oppia.android.app.customview
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.DashPathEffect
+import android.graphics.Paint
+import android.util.AttributeSet
+import android.util.TypedValue
+import android.view.View
+import androidx.core.content.ContextCompat
+import org.oppia.android.R
+
+/**
+ * Custom view to represent chapter progress in [StoryActivity].
+ *
+ * Reference: https://stackoverflow.com/a/27054463
+ */
+class VerticalDashedLineView @JvmOverloads constructor(
+  context: Context,
+  attrs: AttributeSet? = null
+) : View(context, attrs) {
+  private val paint: Paint
+  private lateinit var canvas: Canvas
+
+  init {
+    val dashGap: Float = dpToPx(16)
+    val dashLength: Float = dpToPx(32)
+    val dashThickness: Float = dpToPx(6)
```

Comment:
Shouldn't we declare these values as constants above?

Reply:
Done.